File: backend - (MySQL, Spring)/backend/src/main/java/com/name/backend/model/Item.java
package com.name.backend.model;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@Entity
@Getter
@Setter
public class Item {
    @Id
    @GeneratedValue
    private Long id;
    @Column(nullable = false)
    private String title;
    private String description;
    @Column(nullable = false)
    private boolean finished = false;
    private Date deadline;
}

File: backend - (SQL Server, ASP.NET)/backend/backend/Models/Item.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Models
{
    public class Item
    {
        [Key]
        public int ID { get; set; }
        [Required]
        public string? Title { get; set; }
        public string? Description { get; set; }
        [Required]
        public bool finished { get; set; } = false;
        public DateTime Deadline { get; set; }

    }
}
